src/components/molecules/InformationCard/InformationCard.tsx

```diff
@@ -15,7 +15,7 @@ const InformationCard: React.FC<InformationCardProps> = ({
   className,
 }) => (
   <div className={classNames("information-card-container", className)}>
-    <h4 className="title">{title}</h4>
+    {title && <h4 className="title">{title}</h4>}
```

Comment:
What's the thoughts/motivation for making the title here optional out of curiosity?